ChangeLog

```diff
@@ -1,3 +1,6 @@
+2024-11-25  Simon Guest <simon.guest@tesujimath.org>
```

Comment:
Can you make it two spaces also between your name and your email?

Then one new line, and a tab (or eight spaces) to the *.

Thank you!

Reply:
Done!